jukebox_radio/core/utils.py

```diff
@@ -13,7 +13,7 @@
 
 def generate_spotify_authorization_uri(request):
     domain_prefix = "https" if request.is_secure() else "http"
-    current_site = get_current_site(request)
+    current_site = request.get_host()
```

Comment:
bug exposed once since backend and frontend are coming from different domains. this is the correct way to get the _server domain_.